python/test/t_ProjectionStrategy_std.py

```diff
@@ -0,0 +1,98 @@
+#! /usr/bin/env python
+
+import openturns as ot
+from openturns.usecases import flood_model
+
+ot.TESTPREAMBLE()
+
+ot.Log.Show(ot.Log.NONE)
+
+sampleSize = 500
+totalDegree = 7
+
+#
+print("+ Compute flood model by regression")
```

Comment:
least squares, not regression

Reply:
I agree: thank you for pointing  this.

Reply:
Ok, fixed at 1ba30677e7652bb73d79d2c0d4fe1534bdadc296